jenkins-master/config.xml

```diff
@@ -280,7 +280,7 @@
           </default>
           <int>2</int>
           <string>MOZMILL_AUTOMATION_VERSION</string>
-          <string>2.0.9.1</string>
+          <string>2.0.10</string>
           <string>NOTIFICATION_ADDRESS</string>
           <string></string>
```

Comment:
I asked above as I see there is no address here. I will admit to not really knowing what either of these files are doing, though.